src/executor.cpp

```diff
@@ -561,51 +551,51 @@ void Executor::execute01()
 
 void Executor::execute12()
 {
-    // Need to show blank spaces in case no srcData as function is enabled.
-    constexpr auto blankHexWord = "        ";
-    std::vector<std::string> output(4, blankHexWord);
+    displaySRCFunction(constants::FUNCTION_12);
+}
+
+void Executor::displaySRCFunction(const uint8_t function)
+{
+    // Need to show zeroes in case no srcData as function is enabled.
+    constexpr auto hexZeroes = "00000000";
+    std::vector<std::string> output(4, hexZeroes);
```

Comment:
use TOTAL_RAW_HEX_WORD instead of 4

Reply:
TOTAL_RAW_HEX_WORDS is 8. here we display only 4.
WORDS_PER_DISPLAY = 4 , this suits better..is that fine?